plugins/web/ui/src/routes/status/+page.svelte

```diff
@@ -0,0 +1,105 @@
+<script lang="ts">
+	import { useQuery } from '@sveltestack/svelte-query';
+
+	import Table from '$lib/components/Table.svelte';
+	import Expandable from '$lib/components/Expandable.svelte';
+
+	import { formatBytes } from '$lib/util/formatBytes';
+	import Tooltip from '$lib/components/Tooltip.svelte';
+
+	const dtFormat = new Intl.DateTimeFormat('en-US', {
+		timeStyle: 'medium',
+		timeZone: 'UTC'
+	});
+
+	const getStatus = async () => {
+		try {
+			const API_BASE = 'http://localhost:42001/api/v0';
```

Comment:
Is there a way to write this so that the "installed" version automatically uses its own domain as `API_BASE`, while still allowing development against an external server?

Reply:
Yes, that can be done. Such parameters are usually done via `.env` files. Do you want me to push a commit here?

Reply:
Yes, that would be great

Reply:
Done. To configure the API Endpoint - copy the `.env.example` file to `.env` and edit the value.

Reply:
After trying for 30min, I didn't manage to make the `.env` work:
`pnpm exec vite dev` with `.env` in the same folder. I resorted to manually editing the svelte source file to make this work.

Reply:
I am suspecting this is due to CORS issues. Did you see anything indicating that in the console? Anyway, I have pushed a commit that changes the endpoint from `127.0.0.1` to `localhost` which temporarily seems to fix the CORS issue. Paging @lava about the backend.

Reply:
Hard to say without seeing the exact error, from my understanding of CORS the `Access-Control-Allow-Origin: *` header should allow embedding by arbitrary origins, but maybe this was against an older version of VAST or some browsers are more strict about cross-site requests than others?